features/addressing/managing_countries/province_validation.feature

```diff
@@ -34,3 +34,12 @@ Feature: Province validation
         And I try to save changes
         Then I should be notified that name is required
         And the province should still be named "Northern Ireland" in this country
+
+    @api
```

Comment:
Why we couldn't add the `@api` tag to the previous scenario?

Reply:
in this `api` scenario i use different context (because i operate on sub-resource related to county) so I changed a bit steps name to avoid name conflicts and steps implementation

Reply:
But in all `api` scenarios we are using different contexts than in `ui` scenarios. I would be for adjusting steps names to be able to have one scenario for this feature

Reply:
Should be merged together